Competition2018/src/main/java/competition/subsystems/autonomous/AutonomousDecisionSystem.java

```diff
@@ -170,7 +171,7 @@ public double getDelay() {
     public List<FieldPose> createPathToNearbySwitchPlate() {
         ArrayList<FieldPose> points = new ArrayList<FieldPose>();
         points.add(new FieldPose(new XYPair(0*12, 1.5*12), new ContiguousHeading(90)));
-        points.add(new FieldPose(new XYPair(-6*12, 7.5*12), new ContiguousHeading(90)));
+        points.add(new FieldPose(new XYPair(-6*12, 8*12), new ContiguousHeading(90)));
```

Comment:
This field feels a little different -and I"m thinking we may want to consider moving even further forward in order to be flush, which might reduce that auto thrashing when we change direction.